content/hardware/04.pro/boards/portenta-x8/tutorials/09.image-flashing/content.md

```diff
@@ -13,26 +13,146 @@ hardware:
 
 ## Overview
 
-In this tutorial, you will learn how to manually flash your Portenta X8 with the image provided by Arduino. You will flash your board through USB using the Terminal. The instructions below are meant to be used with a **Windows Operating System**.
+In this tutorial, you will learn the different methods to flash your Portenta X8 with the image provided by Arduino. The process involves using the terminal to perform a USB based flashing procedure on a **Windows Operating System**. By the end, you will be able to update your Portenta X8 to the latest version, ensuring optimal performance and security.
 
 ## Goals
 
-- Learn how to get the required files
-- Learn how to set up the correct structure of the files
-- Learn how to set up the board
-- Learn how to flash the device
+- Understand how to download and organize the required files for the flashing process
```

Comment:
Add also: 
- Learn how to update your Portenta X8 from the Linux Wizard

Reply:
updated!